### ISSUE TITLE
Getting error:  StaticInjectorError(Platform: core)[InjectionToken ToastConfig] in angular 8

### ISSUE THREAD

**Issue by sonusindhu** [scttcper/ngx-toastr#726]
Hi Guyz,

Thank you for such a great package.

I have ngx-toastr package installed on my project but I am not using it right now. so I have no export that module in my app component. Once I run the application I am getting this error:

**core.js:4002 ERROR Error: Uncaught (in promise): NullInjectorError: StaticInjectorError(AppModule)[InjectionToken ToastConfig]: 
  StaticInjectorError(Platform: core)[InjectionToken ToastConfig]: 
    NullInjectorError: No provider for InjectionToken ToastConfig!
NullInjectorError: StaticInjectorError(AppModule)[InjectionToken ToastConfig]: 
  StaticInjectorError(Platform: core)[InjectionToken ToastConfig]:**

Once I import:

`import { ToastrModule } from 'ngx-toastr';`

```
ToastrModule.forRoot({
      timeOut: 5000,
      positionClass: 'toast-bottom-center',
      preventDuplicates: true,
      closeButton: true,
      progressBar: true,
      maxOpened: 1,
      autoDismiss: true,
      enableHtml: true
    }), 
```

then that is fine. As I am planning to use in the future. 
Do I need to import it?

**Comment by scttcper** [comment]
What?

**Comment by shayaneumar-mitel** [comment]
@sonusindhu Are you importing it in the App Module?

@scttcper I have observed that when I import the module in any lazy loaded module I am getting the same error that @sonusindhu  has mentioned. Importing it in the app module works fine.

Does the import work only in AppModule?

**Comment by custompro12** [comment]
I can confirm that I get the same error when attempting to import the ToastrModule inside of a lazy loaded feature module.

**Comment by scttcper** [comment]
Yeah I would assume this module has problems being lazy loaded because of the “entryCompnents”

**Comment by hallowatcher** [comment]
is there any workarounds for this? this is a critical bug atm and I cannot use toasts at all inside lazy loaded modules

**Comment by scttcper** [comment]
does anyone have a small reproduction?

**Comment by ng-model** [comment]
yes the import will only work in `app.module` and not in any lazy loaded modules.

**Comment by Vaaljan** [comment]
Been almost a Year and seems this issue still exists...

**Comment by scttcper** [comment]
@Vaaljan whats the fix?

**Comment by Vaaljan** [comment]
@scttcper I think the problem has to do with the version of nxg-toastr. 
I was using 12.1.0 with Angular 8 which seems incompatible. 
So I downgraded to 11.3.3 on Angular 8

Also had to make sure that the import into the lazy module only happens once like in the AppModule.
